lighthouse-core/lib/manifest-parser.js

```diff
@@ -229,8 +229,14 @@ function parseIcon(raw, manifestUrl) {
     src.value = undefined;
   }
   if (src.value) {
-    // 9.4(4) - construct URL with manifest URL as the base
-    src.value = new URL(src.value, manifestUrl).href;
+    try {
+      // 9.4(4) - construct URL with manifest URL as the base
+      src.value = new URL(src.value, manifestUrl).href;
+    } catch (_) {
+      // 9.6 "This algorithm will return a URL or undefined."
```

Comment:
unfortunately the numbering has changed since https://www.w3.org/TR/2016/WD-appmanifest-20160825/ or whatever was the current draft when the other numbers are written. We probably should have included a base version for that

Reply:
yeah I noticed that, updated the link to point to that version 👍